### PR TITLE
docs: add day2 configuration document

### PR DESCRIPTION
**Pull request by chlins**
Close: https://github.com/goharbor/harbor-operator/issues/562

Signed-off-by: chlins <chlins.zhang@gmail.com>

### CODE REVIEW

File: docs/configurations/day2-config.md [hunk 1]
```diff
@@ -1,3 +1,76 @@
...
+  annotations:
+    # required.
+    # if not define the anno, the config map will not work.
+    # the key is `goharbor.io/configuration`, and the value is your harborcluster cr name.
+    goharbor.io/configuration: harborcluster-sample
+data:
+  # provide your harbor configuration by yaml format.
+  config.yaml: |
+    email_ssl: true
+    email_password: secret-sample # the value is the name of secret which store the email_password.
```
Comment (steven-zou) [inline docs/configurations/day2-config.md:39]: Any way to list all the configuration items here? @chlins 

cc @heww

**Review by steven-zou — COMMENTED** [review]
see my comments

File: docs/configurations/day2-config.md (hunk 1 above)
Comment (chlins) [reply docs/configurations/day2-config.md:39]: Maybe we can link harbor swagger yaml? https://github.com/goharbor/harbor/blob/c701ce09fa2a34b7ea26addb431154f9812de8af/api/v2.0/legacy_swagger.yaml#L516

Comment (steven-zou) [reply docs/configurations/day2-config.md:39]: I think it's ok. @chlins

**Review by steven-zou — APPROVED** [review]
LGTM

**Review by heww — APPROVED** [review]

**Pull request merged**